index.html

```diff
@@ -49,8 +49,187 @@ <h1 class="username">I'm Mark, Glad to see you!</h1>
+                    </div>
+                </div>
+            </div>
+            <div class="card-works">
+                <div class="snapshoot-portfolio">
+                    <img src="./images/yoga.svg" alt="" class="portfolio-snapshoot">
+                </div>
+                <div class="left-block">
+                    <div class="project-summary">
+                        <h1 class="project-title">Tonic</h1>
+                        <div class="summary-frame">
+                            <span class="client">CANOPY</span>
+                            <span class="role">Back End Dev</span>
+                            <span class="year">2015</span>
+                        </div>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="tags">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button class="project-button" type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card-works">
+                <div class="snapshoot-portfolio">
+                    <img src="./images/professional-art-printing.svg" alt="" class="portfolio-snapshoot">
+                </div>
+                <div class="left-block">
+                    <div class="project-summary">
+                        <h1 class="project-title">Multi-Post Stories</h1>
+                        <div class="summary-frame">
+                            <span class="client">CANOPY</span>
+                            <span class="role">Back End Dev</span>
+                            <span class="year">2015</span>
+                        </div>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="tags">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button class="project-button" type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-myself">
+            <div class="skill-top">
+                <div class="name">
+                    <h1 class="username">About Myself</h1>
```

Comment:
- [ ] It is good practice to have only one `<h1>` per page, and that should be for the main title. Kindly change this to any of h2 - h6 tag. You can read more [here](https://www.hobo-web.co.uk/headers/)